inc/saiport.h

```diff
@@ -91,6 +91,60 @@ typedef struct _sai_port_oper_status_notification_t
 
 } sai_port_oper_status_notification_t;
 
+/**
+ * @brief Attribute data for #SAI_PORT_ATTR_ERROR_EVENT
+ */
+typedef enum _sai_port_error_event_t
+{
+    /** Unknown */
+    SAI_PORT_ERROR_EVENT_UNKNOWN,
+
+    /** Remote-fault */
+    SAI_PORT_ERROR_EVENT_REMOTE_FAULT,
+
+    /** Local-fault */
+    SAI_PORT_ERROR_EVENT_LOCAL_FAULT,
+
+    /** Pre-emphasis failed */
+    SAI_PORT_ERROR_EVENT_PREEMPHASIS_FAILED,
```

Comment:
Why should this event be sent via notification ? Or we want after falure on set to send it asynchroneously via notification from SAI to SWSS ?